### PR TITLE
docs: add security faq explaining AES-CTR crypto issues, fixes #5254

### LINKED ISSUE #5254: Borg cryptopgraphy is crtizied - is this an issue for off-site backups?
## Have you checked borgbackup docs, FAQ, and open Github issues?

Yes

## Is this a BUG / ISSUE report or a QUESTION?

Question

## Describe the problem you're observing.

I want to use borg for off-site backup of privacy-critical data. I tried to learn about borg cryptopgraphy using the documentation, #1044, and web resources. I found the following YCombinator post:
https://news.ycombinator.com/item?id=19484167

As a non-expert, I wonder of the points raised in this topic are of relevance and/or adressed in #1044 ?
This blog post caused me not to trust Borg for now for off-site backups, so this might be true for other users as well. It could be desirable to add a borg's developer point-of-view to this blog post?

(I want to use this way too to say thank you to all developers!)

### PR DESCRIPTION
**Pull request by fantasya-pbem**

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -420,6 +421,36 @@ to change them.
...
+  not having multiple clients per repo.
+
+- Even if there is only one client, if client-side information is completely
+  lost (e.g. due to disk defect), the client also needs to trust the value from
+  server side. You can avoid this by not continuing to write to the repository
+  after you have lost clientside borg information.
+
+If a nonce (counter) value is reused, AES-CTR mode crypto is broken. This can
+be avoided by not using AES-CTR mode, but this needs bigger changes and won't
+happen soon.
```
Comment (ThomasWaldmann) [inline docs/faq.rst:452]: "won't happen soon" is relative to now and based on some assumptions.

first will change any day, second might not be true, so i'ld rather leave that away.

Comment (fantasya-pbem) [reply docs/faq.rst]: ok

File: docs/faq.rst [hunk 2]
```diff
@@ -30,7 +30,8 @@ Can I backup from multiple servers into a single repository?
 Yes, this is *possible* from the technical standpoint, but it is
 *not recommended* from the security perspective. BorgBackup is
 built upon a defined :ref:`attack_model` that cannot provide its
-guarantees for multiple clients using the same repository.
+guarantees for multiple clients using the same repository. See
+:ref:`borg_security_critique` for a detailed explaination.
```
Comment (ThomasWaldmann) [inline docs/faq.rst:34]: explanation

**Comment by ThomasWaldmann** [comment]
also change commit comment to "..., fixes #5254" so it automatically closes the ticket.

**Comment by ThomasWaldmann** [comment]
Thanks!

**Pull request merged**